StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1125UnitTests.cs

```diff
@@ -0,0 +1,327 @@
+﻿namespace StyleCop.Analyzers.Test.ReadabilityRules
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using StyleCop.Analyzers.ReadabilityRules;
+    using TestHelper;
+    using Xunit;
+
+    /// <summary>
+    /// This class contains unit tests for <see cref="SA1125UseShorthandForNullableTypes"/>.
+    /// </summary>
+    public class SA1125UnitTests : DiagnosticVerifier
+    {
+        /// <summary>
+        /// This is a regression test for DotNetAnalyzers/StyleCopAnalyzers#385.
+        /// <see href="https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/385">SA1125
+        /// UseShorthandForNullableTypes incorrectly reported in XML comment</see>
+        /// </summary>
+        [Theory]
+        [InlineData("Nullable{T}", "Nullable{T}")]
+        [InlineData("System.Nullable{T}", "System.Nullable{T}")]
+        [InlineData("global::System.Nullable{T}", "global::System.Nullable{T}")]
+        public async Task TestSeeAlsoNullable(string longForm, string shortForm)
```

Comment:
Is there a reason why you included shortForm here? It isnt used in the test (there are a few occurences of this in this file)